cli/trampolines/trampolines_aarch64.S

```diff
@@ -0,0 +1,11 @@
+#include "../../src/jl_exported_funcs.inc"
+
+#define XX(name) \
+.global name; \
+name##:; \
```

Comment:
@Keno do you happen to already know any good references of the `.cfi_startproc` and such that we should have here to preserve `@profile` capability when inside these frames. We can probably derive them by looking at the output of clang/gcc for these different targets, if not.